platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -127,4 +127,5 @@ public record PlatformSchedulersConfig(
         @ConfigProperty(defaultValue = "SEQUENTIAL") TaskSchedulerType futureEventBufferSchedulerType,
         @ConfigProperty(defaultValue = "500") int futureEventBufferUnhandledCapacity,
         @ConfigProperty(defaultValue = "SEQUENTIAL") TaskSchedulerType issDetectorSchedulerType,
+        @ConfigProperty(defaultValue = "100") int hashLoggerUnhandledTaskCapacity,
         @ConfigProperty(defaultValue = "500") int issDetectorUnhandledCapacity) {}
```

Comment:
nit: this can be done in a following PR, but there are 2 vars related to the iss detector that have been split by the new value

Reply:
Since there are other comments to still resolve, I think this should also be fixed in this PR.

Reply:
fixed.